source/cl_main.c

```diff
@@ -880,7 +881,8 @@ void CL_RelinkEntities (void)
 	        dl->color[0] = 0;
 			dl->color[1] = 255;
 			dl->color[2] = 0;
-	        dl->type = SetDlightColor (2, lt_rocket, true);
+			//DLight color 4 is green
+	        dl->type = SetDlightColor (4, lt_rocket, true);
```

Comment:
I figured it would make sense to set it to the color expected in function "SetDlightColor". If not, what is the purpose of setting this at all?

Reply:
`SetDlightColor` sounds legacy. Can you investigate what exactly this does? It probably can be fully deprecated, since we do just edit the actual color values of the object directly.

Reply:
Looked into this and confirmed it is legacy. I removed all references to this and confirmed that it introduced no issues.